src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/reporting/serviceprovider/performance/PerformanceReportJobConfiguration.kt

```diff
@@ -45,6 +49,15 @@ class PerformanceReportJobConfiguration(
   ): HibernateCursorItemReader<Referral> {
     // this reader returns referral entities which need processing for the report.
     val contracts = serviceProviderAccessScopeMapper.fromUser(AuthUser(userId, authSource, userName)).contracts
+
+    logger.info(
```

Comment:
i really wish we could pass a list of contract refs to the job. then we could just use spring batch's logging to tell us the job params.

Reply:
we could just comma seperate them. but as @andrewburrowsmoj has mentioned before, we have a 250 char limit. could there ever be that many contracts for one user that it would overflow it??